### PR TITLE
[USER_SETTINGS] added

### PR DESCRIPTION
**Pull request by AntonNazarenko**
add user settings config
add user settings default values
add docs

### CODE REVIEW

**Comment by JacobPlaster** [comment]
Lets add a changelog as well, like this: https://github.com/bitfinexcom/bfx-hf-algo/blob/master/CHANGELOG

File: README.md [hunk 1]
```diff
@@ -1 +1,72 @@
...
+
+const isAuthorized =  require('../../util/ws/is_authorized')
+
+  
+
+module.exports  =  async  (server, ws, msg)  =>  {
+
+	const  {  db  }  =  server
+
+	const  {  UserSettings  }  =  db
```
Comment (f3rno) [inline README.md]: Please fix the spacing above and below, there are many redundant newlines.

File: docs/user_settings.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+
+<a name="UserSettings"></a>
+
+## UserSettings
+The UserSettings config provides a set of charts, themes and default values
+manages lifetime events/order execution. Internally it hosts a Manager
```
Comment (f3rno) [inline docs/user_settings.md]: This needs to be fixed :slightly_smiling_face:

File: package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "bfx-hf-ui-config",
+  "version": "1.0.0",
+  "engines": {
+    "node": ">=6"
+  },
+  "description": "",
```
Comment (f3rno) [inline package.json]: Fill in `description`

**Review by f3rno — CHANGES_REQUESTED** [review]
Some minor changes, but looks good :+1:

File: package.json (hunk 3 above)
Comment (AntonNazarenko) [reply package.json]: done

File: docs/user_settings.md (hunk 2 above)
Comment (AntonNazarenko) [reply docs/user_settings.md]: done

File: README.md (hunk 1 above)
Comment (AntonNazarenko) [reply README.md]: done

**Pull request merged**